src/tests/tests/firstparties.js

```diff
@@ -164,4 +165,38 @@ QUnit.test('google search de-instrumentation', (assert) => {
   fixture.appendChild(util_script);
 });
 
+QUnit.test('tumblr link unwrapping', (assert) => {
```

Comment:
We probably don't need a unit test for this PR, especially if we reuse `google-static.js` for Tumblr.